packages/riipen-ui/src/components/ButtonBase.jsx

```diff
@@ -0,0 +1,84 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+const ButtonBase = ({
+  ariaLabel,
+  classes,
+  children,
+  type,
+  variant,
+  ...other
+}) => {
+  const className = clsx("button", variant, classes);
+  return (
+    <>
+      <button
+        aria-label={ariaLabel}
+        className={className}
+        type={type}
+        {...other}
+      >
+        {children}
+      </button>
+      <style jsx>{`
+        .basic {
```

Comment:
I wonder if we can just remove basic css class since it's empty. What is the difference between default and basic?

Reply:
`basic` would make the wrapped children look like a basic html `button`
`default` removes the styles of standard html `button` element so that this component becomes an unstyled wrapper for its children